cms/djangoapps/contentstore/views/course.py

```diff
@@ -999,7 +999,6 @@ def update_tab(tabs, tab_type, tab_enabled):
         """
         tab_panel = {
             "type": tab_type.name,
-            "name": tab_type.title,
         }
         has_tab = tab_panel in tabs
```

Comment:
I'm confused how this would ever return `True`. Like, doesn't the courseware tab look like 
```json
{
  "type": "courseware",
  "name": "Courseware"
}
```